cluster/aws/util.sh

```diff
@@ -36,7 +36,9 @@ source "${KUBE_ROOT}/cluster/aws/${KUBE_CONFIG_FILE-"config-default.sh"}"
 source "${KUBE_ROOT}/cluster/common.sh"
 source "${KUBE_ROOT}/cluster/lib/util.sh"
 
-ALLOCATE_NODE_CIDRS=true
+if [[ "${NETWORK_PROVIDER:-}" != "cni" ]]; then
```

Comment:
There is no mention of such a dependency at https://github.com/containernetworking/cni/blob/master/Documentation/flannel.md

CNI is used by more than just Kubernetes; I guess it's possible that someone has built a Kubernetes-specific CNI plugin, but I'm prepared to worry about that when I see it.